internal/olm/operator/registry/fbcindex/fbc_registry_pod.go

```diff
@@ -233,8 +237,11 @@ func (f *FBCRegistryPod) podForBundleRegistry(cs *v1alpha1.CatalogSource) (*core
 		return nil, fmt.Errorf("configMap error: %w", err)
 	}
 
-	volumes := []corev1.Volume{}
-	volumeMounts := []corev1.VolumeMount{}
+	var (
+		volumes            []corev1.Volume
+		sharedVolumeMounts []corev1.VolumeMount
+		gzipVolumeMount    []corev1.VolumeMount
+	)
```

Comment:
Nit: I *personally* prefer to do something like:
```go
volumes := []corev1.Volume{}
sharedVolumeMounts := []corev1.VolumeMount{}
gzipVolumeMounts := []corev1.VolumeMount{}
```

This shouldn't block the merge though if everything else checks out.

Reply:
I don't have a strong opinion about it, but according to the golang wiki, this format is prefered: https://github.com/golang/go/wiki/CodeReviewComments#declaring-empty-slices.